src/common/util.h

```diff
@@ -403,15 +403,6 @@ void oscap_strtoupper(char *str);
 bool oscap_ptr_cmp(void *node1, void *node2);
 
 /**
- * find file with given name and mode in given paths
```

Comment:
It was removed in 71da0a6e96a913dcf66ece70c1b78da9d7fcf26d 5 years ago, so I guess we don't need it :D